DVTShowcase/LoginViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  LoginViewController.swift
+//  DVTShowcase
+//
+//  Created by Idris SOP on 2017/09/18.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+import UIKit
+
+class LoginViewController: DVTShowcaseViewController, UITextFieldDelegate {
+    @IBOutlet var usernameTF: UITextField!
```

Comment:
There is some debate around making IBOutlets weak or strong. Generally though it's considered best practice to make all IBOutlets weak, unless there is some specific reason for making them strong. The reason they should be weak is because the nib already holds a strong reference to them. Making them strong could ( under some circumstances ) lead to memory reference cycles and hence a memory leak.